css/main.css

```diff
@@ -8,3 +8,26 @@ body {
   color: #3b3d40;
   font-size: 14px;
 }
+header{
```

Comment:
usar classe no lugar dessa tag,
tipo assim: 
```suggestion
.header{
```

Reply:
belezaaa !! pode deixarrr